src/TConsole.cpp

```diff
@@ -1040,7 +1044,7 @@ void TConsole::slot_toggleLogging()
 
 void TConsole::slot_toggleReplayRecording()
 {
-    if (mIsDebugConsole) {
+    if (mType & CentralDebugConsole) {
```

Comment:
Surely this should include everything apart from the `MainConsole` so that only that type actually does anything here.